src/Apps/Apps.jl

```diff
@@ -38,6 +71,18 @@ function overwrite_file_if_different(file, content)
     end
 end
 
+function check_apps_in_path(apps)
+    for app_name in keys(apps)
+        if Sys.which(app_name) === nothing
```

Comment:
Perhaps also warn if it points to another executable too?

Reply:
Yeah, makes sense

Reply:
Added in last commit